src/covid19sim/configs/config.py

```diff
@@ -299,61 +293,26 @@
 ENVIRONMENTAL_INFECTION_KNOB = 0.0005
 
 ## INTERVENTIONS
-BIG_NUMBER = 10000000
 HYGIENE_EFFECT = 0.2
 
 # TRACKER
 EFFECTIVE_R_WINDOW = 10 # days
-GET_RISK_PREDICTOR_METRICS = False # compute risk but not modify the behavior
 
 # MASK
 MASK_EFFICACY_NORMIE = 0.32
 MASK_EFFICACY_HEALTHWORKER = 0.98
 BASELINE_P_MASK = 0.5
-MASKS_SUPPLY = BIG_NUMBER
+MASKS_SUPPLY = 1000000
 
-## INTERVENTIONS
-BIG_NUMBER = 10000000
-INTERVENTION_DAY = -1 # <0 no interventions
-INTERVENTION = "Tracing"
-PERCENT_FOLLOW = 1.0
-P_HAS_APP = 0.5
```

Comment:
It might be less confusing to have `P_HAS_APP` and `PERCENT_FOLLOW` in `config.py` as it is more of a demographics parameter.

Reply:
On a second thought, it seems like these should be in `exp_config.yml`